sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1649,15 +1635,26 @@ static bool setEnvVar(const char *name, const char *value) {
   return true;
 }
 
-static class ZeUSMImport {
+static class ZeUSMImportExtension {
   // Pointers to functions that import/release host memory into USM
   ze_result_t (*zexDriverImportExternalPointer)(ze_driver_handle_t hDriver,
                                                 void *, size_t);
   ze_result_t (*zexDriverReleaseImportedPointer)(ze_driver_handle_t, void *);
-  bool ImportSupported;
 
 public:
-  ZeUSMImport() : ImportSupported{false} {}
+  // Whether env var SYCL_USM_HOSTPTR_IMPORT has been set requesting
+  // host ptr import during buffer creation.
+  bool ImportRequested;
```

Comment:
Why do you need this one? Why not have single "bool" to mean both requested an available?